### PR TITLE
feat(preview-table): create data-table preview markup (without style) & apply load API 

### PR DESCRIPTION
**Pull request by piggggggggy**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [ ] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA

image

### CODE REVIEW

File: apps/web/src/common/modules/widgets/_components/WidgetFormOverlayPreviewTable.vue [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+    previewData: computed(() => widgetGenerateState.previewData),
+    selectedDataTableId: computed(() => widgetGenerateState.selectedDataTableId),
+    selectedDataTable: computed(() => widgetGenrateGetters.selectedDataTable),
+});
+
+const state = reactive({
+    loading: false,
+    data: undefined,
+    labelFields: computed<string[]>(() => Object.keys(storeState.selectedDataTable?.labels_info ?? {})),
+    dataFields: computed<string[]>(() => Object.keys(storeState.selectedDataTable?.data_info ?? {})),
```
Comment (sulmoJ) [inline apps/web/src/common/modules/widgets/_components/WidgetFormOverlayPreviewTable.vue:35]: Name of fields changed.
Label Field -> Label Info
Data Field -> Data Info

File: apps/web/src/common/modules/widgets/_components/WidgetFormOverlayPreviewTable.vue [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+    fields: computed<DataTableFieldType[]>(() => {
+        console.debug('fields', [
+            ...state.labelFields,
+            ...state.dataFields,
+        ]);
```
Comment (sulmoJ) [inline apps/web/src/common/modules/widgets/_components/WidgetFormOverlayPreviewTable.vue]: 👀 👀 👀

File: apps/web/src/common/modules/widgets/_components/WidgetFormOverlayPreviewTable.vue [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+            type: 'item',
+            name: GRANULARITY.YEARLY,
+            label: 'Yearly',
+        },
+    ])),
+});
+
+/* Events */
+const handleSelectGranularity = async (granularity: Granularity) => {
+    console.debug('handleSelectGranularity', granularity);
```
Comment (sulmoJ) [inline apps/web/src/common/modules/widgets/_components/WidgetFormOverlayPreviewTable.vue]: 🙄 🙄

**Review by sulmoJ — APPROVED** [review]
GOOOOOD

File: apps/web/src/common/modules/widgets/_components/WidgetFormOverlayPreviewTable.vue (hunk 2 above)
Comment (yuda110) [inline apps/web/src/common/modules/widgets/_components/WidgetFormOverlayPreviewTable.vue]: quibble: console.debug!

**Review by yuda110 — APPROVED** [review]
LGTM~

**Pull request merged**